src/main/java/org/folio/dcb/service/impl/BaseLibraryService.java

```diff
@@ -4,11 +4,15 @@
 import lombok.extern.log4j.Log4j2;
 import org.folio.dcb.domain.dto.*;
```

Comment:
fix the wildcard usage pls

Reply:
Removed from this file and from other files where wildcard imports are used.